### PR TITLE
Fix typo in ingestion spec sample

### LINKED ISSUE #11432: Typo in Ingestion Spec Sample
### Affected Version

0.21.1

### Description

URL: https://druid.apache.org/docs/latest/ingestion/index.html
Source file: https://github.com/apache/druid/blob/master/docs/ingestion/index.md

### PR DESCRIPTION
**Pull request by jerryleooo**
Fixes #11432 

### Description

Fix typo in the ingestion spec sample

### CODE REVIEW

File: docs/ingestion/index.md [hunk 1]
```diff
@@ -309,8 +309,8 @@ Example ingestion spec for task type `index_parallel` (native batch):
       },
       "dimensionsSpec": {
         "dimensions": [
-          { "type": "string", "page" },
-          { "type": "string", "language" },
+          { "type": "string", "name": "page" },
```
Comment (FrankChen021) [inline docs/ingestion/index.md:312]: I think we could use "page" directly instead of an json object. See Line 448 and 472 of this file.

**Comment by FrankChen021** [comment]
Thanks for your contribution. I see there are multiple occurrences of this bad typo, kindly search and fix all of them.

File: docs/ingestion/index.md (hunk 1 above)
Comment (clintropolis) [reply docs/ingestion/index.md:312]: lines 385-386, 663-664 also have this same mistake

Comment (jerryleooo) [reply docs/ingestion/index.md]: Thanks @clintropolis , I have fixed them all.

**Review by FrankChen021 — APPROVED** [review]
👍

**Review by suneet-s — APPROVED** [review]

**Comment by suneet-s** [comment]
Thanks for the contribution @jerryleooo !

**Pull request merged**